templates/Project/Etude/TabVoir/Redaction.html.twig

```diff
@@ -386,7 +386,13 @@
                 <td>
                     <a href="{{ path('project_procesverbal_modifier', {'id': etude.pvr.id}) }}">{{ 'suivi.pvr'|trans({}, 'project') }}</a>
                 </td>
-                <td>{{ 'suivi.toutes'|trans({}, 'project') }}</td>
+                <td>
+                {%for phase in etude.pvr.phases %}
+                    {{ phase.titre}} <br>
+                {% else %}
+                    {{ 'suivi.aucune'|trans({}, 'project') }} !
+                {% endfor %}
+                    </td>
```

Comment:
Erreur d'indentation. La boucle for devrait etre indentée d'un niveau et le `</td>` de moins un niveau